src/redux/crypto/crypto.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable no-param-reassign */
```

Comment:
As stated by the previous reviewer, kindly ensure that linters are not dissabled in order to help you achievet the best standard practice at all time. This `no-param-reassign` rule aims to prevent unintended behavior caused by modification or reassignment of function parameters. Kindly check [this link](https://eslint.org/docs/latest/rules/no-param-reassign) for further clarification. :+1: